.env

```diff
@@ -0,0 +1,13 @@
+# Local development database settings
+
```

Comment:
So it is a violation! In fact you could get fired for this! Just by pushing your `.env` file to an open source platform such as github.